### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: aura/lib/Aura/Languages.hs [hunk 1]
```diff
@@ -455,7 +460,7 @@ getRealPkgConflicts_1 (bt . pnName -> prnt) (bt . pnName -> p) (bt -> r) (bt ->
     Indonesia  -> "Dependensi " <> p <> " meminta versi " <> d <> " namun versi paling baru adalah " <> r <> "."
     Chinese    -> "依赖 " <> p <> " 需要版本 " <> d <> "，但是最新的版本是 " <> r <> "。"
     Esperanto  -> "La pakaĵo, " <> prnt <> ", dependas de versio " <> d <> " de " <> p <> ", sed la plej nova versio estas " <> r <> "."
-    Dutch      -> "Het pakket " <> prnt <> ", hangt af van versie " <> d <> " van " <> p <> ", maar de meest recente versie is " <> r <> "."
+    Dutch      -> " <> prnt <> " is afhankelijk van versie " <> d <> " van " <> p <> ", maar de nieuwste versie is " <> r <> "."
```
Comment (fosskers) [inline aura/lib/Aura/Languages.hs:463]: I think there is a missing quotation here.

File: aura/lib/Aura/Languages.hs [hunk 2]
```diff
@@ -516,7 +521,7 @@ depError l (BrokenProvides (PkgName pkg) (Provides (PkgName pro)) (PkgName n)) =
   Russian    -> "Пакету " <> bt pkg <> " требуется " <> bt n <> ", предоставляющий " <> bt pro <> "."
   Esperanto  -> "La pakaĵo, " <> bt pkg <> " bezonas " <> bt n <> ", kiu donas " <> bt pro <> "."
   Italian    -> "Il pacchetto " <> bt pkg <> " ha bisogno di " <> bt n <> ", che rende disponibile " <> bt pro <> "."
-  Dutch      -> "Het pakket" <> bt pkg <> " heeft " <> bt n <> " nodig, die " <> bt pro <> " biedt."
+  Dutch      -> " <> bt pkg <> " is afhankelijk van " <> bt n <> ", wat " <> bt pro <> " bevat."
```
Comment (fosskers) [inline aura/lib/Aura/Languages.hs:524]: Here too.

**Review by fosskers — CHANGES_REQUESTED** [review]
Thank you very much!

File: aura/lib/Aura/Languages.hs (hunk 1 above)
Comment (Vistaus) [reply aura/lib/Aura/Languages.hs]: Good catch! I corrected it now. Thanks! 🙂

File: aura/lib/Aura/Languages.hs (hunk 2 above)
Comment (Vistaus) [reply aura/lib/Aura/Languages.hs]: Corrected this too, thanks!

**Comment by fosskers** [comment]
Github is displaying the merge status incorrectly, but this is indeed merged. Thank you very much!

**Pull request closed without merging**